packages/patternfly-3/patternfly-react/src/components/DateTimePicker/DateComponents/DecadeViewHeader.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from '../../../common/helpers';
+
+export const DecadeViewHeader = ({ currDecade, getPrevDecade, getNextDecade }) => (
+  <thead>
+    <tr>
+      <th className="prev" onClick={getPrevDecade}>
+        <span className="glyphicon glyphicon-chevron-left" title="Previous Decade" />
```

Comment:
I think all titles should be accessible (because of localization) to a consumer.

Reply:
titles were used as tool tips and are now removed